content-blocker-lib-ios/ContentBlockerGen/.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```

Comment:
I think the gitignore is missing an entry, as `.swiftpm` isn't supposed to be committed.

Reply:
Updated gitignore